pages/linux/envsubst.md

```diff
@@ -0,0 +1,20 @@
+# envsubst
+
+> Substitutes shell format strings with environment variables in text.
+> Strings to be replaced should be in either `${var}` or `$var` format.
+
+- Replace environment variables in stdin and output to stdout:
+
+`echo "{{\$HOME}}" | envsubst`
```

Comment:
You can use single quotes to prevent shell from interpreting the variable, hence simplifying to - `echo '{{$HOME}}'`

Reply:
Updated, not sure why I didn't think of that. :+1: